Library/Formula/ratfor.rb

```diff
@@ -0,0 +1,15 @@
+class Ratfor < Formula
+  version "1.02"
+  desc "Rational Fortran"
+  homepage "http://www.dgate.org/ratfor/"
+  url "http://www.dgate.org/ratfor/ratfor.tar.gz"
```

Comment:
You'll need to ask upstream for a versioned one, or host a tap for this yourself. Static URLs are bad UX, sadly.